corehq/apps/app_manager/templates/app_manager/form_designer.html

```diff
@@ -50,7 +50,7 @@
 {% block form-view %}
     {% initial_page_data 'CKEDITOR_BASEPATH' CKEDITOR_BASEPATH|static %}
     {% initial_page_data 'days_since_created' request.couch_user.days_since_created %}
-    {% initial_page_data 'form_name' form.name|trans:app.langs %}
+    {% initial_page_data 'form_name' form.name|trans:langs %}
```

Comment:
Aren't `langs` and `app.langs` the same? https://github.com/dimagi/commcare-hq/blob/33e429969c1ee8b53241f6ad00adb4944826e104/corehq/apps/app_manager/views/formdesigner.py#L228

Reply:
You linked me to what would I believe in the template be referenced as `vellum_options.javaRosa.langs` or something like that. `langs` is set in `get_apps_base_context` here: https://github.com/dimagi/commcare-hq/blob/33e429969c1ee8b53241f6ad00adb4944826e104/corehq/apps/app_manager/views/apps.py#L329 and is gotten from `get_langs` which places the currently selected language at the beginning of the list: https://github.com/dimagi/commcare-hq/blob/33e429969c1ee8b53241f6ad00adb4944826e104/corehq/apps/app_manager/views/utils.py#L114. That last bit is the main difference between `langs` and `app.langs`.